hellosign_sdk/hsclient.py

```diff
@@ -1906,6 +1921,13 @@ def _send_signature_request(self, test_mode=False, client_id=None, files=None,
 
             is_for_embedded_signing (bool): send_signature_request and send_signature_request_embedded share the same sending logic. To differenciate the two calls embedded requests are now flagged.
 
+            attachments (list of dict):            A list of attachments, which each has the following attributes:
```

Comment:
Can you update this block with the suggestions from send_signature_request?

Reply:
@oconnor-sn can you address this please. It should be `A list of attachments, each with the following attributes:`